pkg/cmd/kafka/list/list.go

```diff
@@ -164,7 +165,8 @@ func mapResponseItemsToRows(kafkas []kafkamgmtclient.KafkaRequest, selectedId st
 
 		var customerCloud string
 		if id, ok := k.GetClusterIdOk(); ok {
-			customerCloud = *id
+			cluster := (*clusterIdMap)[*id]
```

Comment:
This is not printing out a cluster ID for me, just a pair of emptys parenthesis

Reply:
it does work now since the last push, I'm wondering if this is the best way to display it? I get it makes sense during an interactive selection prompt, but since we already have a table with one value for column then I think we should stick to that. i.e. have a `Hybrid Cluster Name` and a `Hybrid Cluster ID`

Reply:
Yeah that is an interesting point, what would we put there, and I am thinking if all you have is normal kafkas then two columns of info that doesn't apply might be messy

Reply:
@dimakis and I will get back to this conversation around formats and naming in the future